src/components/modal/v2/parts/views/ShortTerm/styles.scss

```
@@ -1,56 +1,129 @@
 @import '../../../styles/globals/colors';
 @import '../../../styles/globals/mixins';
+@import '../../../styles/globals/variables';
 
-.donuts {
-    display: flex;
-    align-items: center;
-    width: 100%;
-
-    @include desktop {
-        min-height: 5em;
-        top: 0;
-        margin-bottom: 0px;
-    }
-
-    &__container {
+.content__body {
+    .donuts {
         display: flex;
         align-items: center;
-        justify-content: space-evenly;
         width: 100%;
-        max-width: 360px;
-        margin: 0 auto;
-        padding: 0 20px 15px 20px;
 
         @include desktop {
-            transform: translate(0px, 1rem);
-            position: relative;
-            top: 30px;
+            min-height: 5em;
+            top: 0;
+            margin-bottom: 0px;
         }
 
+        &__container {
+            display: flex;
+            align-items: center;
+            justify-content: space-evenly;
+            width: 100%;
+            max-width: 360px;
+            margin: 0 auto;
+            padding: 0 20px 15px 20px;
+
+            @include desktop {
+                transform: translate(0px, 1rem);
+                position: relative;
+                top: 30px;
+            }
+
+            @include tablet {
+                max-width: 400px;
+            }
+
+            @include mobile {
+                padding: 0 0 15px;
+            }
+
+            @include smallMobile {
+                padding: 0px;
+            }
+        }
+    }
+    .learnMoreLink__container {
+        font-size: 14px;
+
         @include tablet {
-            max-width: 400px;
+            margin-left: 10px;
```

```suggestion
            margin-top: 10px;
```
In my original suggestion I made a typo and I actually meant `margin-top`. That's my bad!